.github/workflows/build.yml

```
@@ -0,0 +1,122 @@
+        cache: 'npm'
+
+    - name: Install dependencies
+      run: npm ci
+
+    # - name: Install Playwright browsers
+    #   run: npx playwright install --with-deps
+
+    - name: Run E2E tests
+      run: npm run test:e2e
+      env:
+        VITE_API_URL: http://localhost:${{ job.services.backend.ports[80] }}
```

actionlint (1.7.7)

property access of object must be type of string but got "number"

(expression)